python/lsst/ts/hvac/hvac_csc.py

```diff
@@ -249,12 +269,32 @@ async def handle_summary_state(self):
     async def configure(self, config):
         self.config = config
 
+    def _get_topic_enabled_state(self, topic):
+        enabled = False
+        deviceId_index = 0
+        hvac_topic = HvacTopic(topic)
+        device_id = DeviceId[hvac_topic.name]
+        deviceId_index = list(DeviceId).index(device_id)
+
+        if topic in TOPICS_ALWAYS_ENABLED:
+            enabled = True
+        elif hvac_topic.name in TOPICS_WITHOUT_COMANDO_ENCENDIDO:
+            enabled = self.hvac_state[topic]["ESTADO_FUNCIONAMIENTO"].recent_values[-1]
+        else:
+            enabled = self.hvac_state[topic]["COMANDO_ENCENDIDO"].recent_values[-1]
+
+        return deviceId_index, enabled
+
     def _compute_statistics_and_send_telemetry(self):
         self.log.info(
             f"{HVAC_STATE_TRACK_PERIOD} seconds have passed since the last "
             f"computation of the medians, so computing now."
         )
+        enabled_mask = 0b0
         for topic in self.hvac_state:
+            deviceId_index, enabled = self._get_topic_enabled_state(topic)
+            if enabled:
+                enabled_mask += 2 ** deviceId_index
```

Comment:
If not enabled then should you skip this topic by adding a `else: continue`? It seems odd to output median for a topic that is not enabled.

Reply:
Some devices publish telemetry even if they are not enabled so the rest is still necessary.